### PR TITLE
Default File manager

### PR DESCRIPTION
**Pull request by onesounds**
#730

### CODE REVIEW

**Comment by taooceros** [comment]
I will merge #767 first for fody

**Comment by onesounds** [comment]
add arguments of files / total commander / directory opus. 
It seems that it cannot be tested with a plug-in yet, so it was tested with a command line.
I don't know how to run rx explorer as a command line.

This item is likely to be added/changed in the future. When I tested, there was a problem that ended when the data changed slightly. It seems that there is a need for a way to be compatible with the old version of data or reset if the data does not fit. The web search plug-in has the same problem, so this kind of setting seems to require a separate solution


**References**
https://resource.dopus.com/t/arguments-or-parameters-for-windows-shortcuts/5099
https://www.voidtools.com/forum/viewtopic.php?t=6617
https://www.ghisler.ch/wiki/index.php/Command_line_parameters
https://github.com/files-community/Files/pull/6558

**Comment by taooceros** [comment]
Done a rebase to clean some history. Nothing actually change in code

File: Flow.Launcher.Plugin/Interfaces/IPublicAPI.cs [hunk 1]
```diff
@@ -190,5 +190,12 @@ public interface IPublicAPI
         /// <typeparam name="T">Type for Serialization</typeparam>
         /// <returns></returns>
         void SaveSettingJsonStorage<T>() where T : new();
+
+        /// <summary>
+        /// Open Directory in explorer configured by user
```
Comment (jjw24) [inline Flow.Launcher.Plugin/Interfaces/IPublicAPI.cs:195]:
```suggestion
        /// Open directory in an explorer configured by user via Flow's Settings. The default is Windows Explorer
```

File: Flow.Launcher/SelectFileManagerWindow.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+
+namespace Flow.Launcher
+{
+    /// <summary>
+    /// SelectFileManagerWindow.xaml에 대한 상호 작용 논리
+    /// </summary>
```
Comment (jjw24) [inline Flow.Launcher/SelectFileManagerWindow.xaml.cs:23]:
```suggestion
```

**Review by jjw24 — DISMISSED** [review]

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
@onesounds got conflict need to resolve for this one

**Comment by onesounds** [comment]
> @onesounds got conflict need to resolve for this one

Done.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**